client/python/samples/utils/vehicle_utils.py

```diff
@@ -27,7 +27,12 @@ def vehicle_postprocess(response, image_path, output_name, output_save_path):
     confidence_threshold = 0.5
     for i in range(output.shape[2]):
         image = draw_bounding_box(output[0, 0, i, ...], image, width, height, confidence_threshold)
-    cv2.imwrite(os.path.join(output_save_path, 'vehicle-detection' + '.jpg'), image)
+    if not os.path.exists(output_save_path):
+        print("Making directory for output: " + output_save_path)
+        os.makedirs(output_save_path)
+    output_file = os.path.join(output_save_path, 'vehicle-detection' + '.jpg')
```

Comment:
Why is this file name hardcoded? What if there are 2 or more images in the source directory?